sentry-core/src/main/java/io/sentry/core/protocol/Device.java

```diff
@@ -12,10 +12,13 @@
   private String family;
   private String model;
   private String modelId;
-  private String architecture;
-  private Short batteryLevel;
-  private Boolean isCharging;
-  private Boolean isOnline;
+
+  @Deprecated private String architecture; // it was only arch, did we change it?
+
+  private String[] architectures;
+  private Float batteryLevel;
+  private Boolean isCharging; // charging
```

Comment:
New fields have to be added to the protocol (change semaphore, docs, other SDKs eventually)

Reply:
yes, so far only architectures and (screenWidthPixels, screenHeightPixels)

Reply:
Cool. Would be nice to get feedback from others on this. @HazAT hasn't got added his review yet

Reply:
Just fyi, adding new fields here shouldn't be a blocker it will just show up in the UI unformatted.